lib/OpenCloud/ObjectStore/Resource/AbstractResource.php

```diff
@@ -36,7 +38,7 @@
     /** @var string The FQCN of the metadata object used for the container. */
     protected $metadataClass = 'OpenCloud\\Common\\Metadata';
 
-    /** @var \OpenCloud\Common\Service\ServiceInterface The service object. */
```

Comment:
Hmm okay. If there are missing methods, we should add the concrete version in. My head's in [this codebase](https://github.com/php-opencloud/openstack) so I forgot the old SDK's a bit inconsistent at times